### PR TITLE
Update Licensing.md: minor wording updates

### PR DESCRIPTION
**Pull request by jlovejoy**
Fixed a couple missing words or other such clarifications.

Re-wrote the Summary to be less definitive regarding license compliance

Signed-off-by: Jilayne Lovejoy

### CODE REVIEW

**Comment by jlovejoy** [comment]
@swinslow - please have a look. There are a couple other things that I think could be improved, including the use of "conclusion" in relation to no data, but I need to think more of a way to clarify that.

Also we have "reasonable objective" sometimes and "reasonably objective" sometimes - thinking it should be consistently the latter?

File: model/Licensing/Licensing.md [hunk 1]
```diff
@@ -45,11 +45,11 @@ types of Software Artifacts. For example:
     different File (e.g., comment at top of File if it is not within the
     Snippet, LICENSE file in the top directory of a repository)
 
-A declaredLicense is related to a NoneLicenseindicates that the
+A declaredLicense is related to a NoneLicense and indicates that the
```
Comment (swinslow) [inline model/Licensing/Licensing.md:48]: I think this should say "A declaredLicense relationship to a NoneLicense indicates..."

"related" here is (I assume) referring to the use of a Relationship to the NoneLicense value.

File: model/Licensing/Licensing.md [hunk 2]
```diff
@@ -45,11 +45,11 @@ types of Software Artifacts. For example:
...
 corresponding Package, File or Snippet contains no license information
 whatsoever.
 
-A declaredLicense related to aNoAssertionLicense
+A declaredLicense related to a NoAssertionLicense and
```
Comment (swinslow) [inline model/Licensing/Licensing.md:52]: Same thing here, "A declaredLicense relationship to a NoAssertionLicense indicates..."

File: model/Licensing/Licensing.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-The Licensing Profile defines a minimum set of license information required to comply with typical license compliance use cases.
+The Licensing Profile defines a minimum set of license information to facilitate compliance with typical license  use cases.
```
Comment (swinslow) [inline model/Licensing/Licensing.md:7]: With "compliance" removed, I think there's now two spaces between "license" and "use cases" -- probably remove the stray space too?

**Review by swinslow — COMMENTED** [review]
Hi @jlovejoy, I had a couple of suggested tweaks to the edits here -- take a look and see what you think. Otherwise LGTM. Thanks!

**Review by goneall — APPROVED** [review]
LGTM

**Comment by swinslow** [comment]
@jlovejoy Are you good with the further tweaks I made in 5ae17e3?

These were to address the comments above, as well as trying to avoid overloading the term "conclusion" given the presence of the `concludedLicense` relationship.

If these look good to you, then @goneall can merge once he's good with these as well. Thanks!

**Comment by zvr** [comment]
Minor wording comment: it shouldn't be "relationship to a NoneLicense" (or "NoAssertionLicense"), but "relationship to NoneLicense" (without the "a"). Both `NoneLicense` and `NoAssertionLicense` are defined individuals, there is only one of them, so the indefinite article "a" does not make sense.

More substantial comment: I know all examples talk about Packages, Files and Snippets, and the text talks about "Software Artifacts." However, this is not "SoftwareLicensing" profile and I'm pretty certain people will want to use it for non-software artifacts. Do we want to include some wording to this effect (or, conversely, explicitly mention that this is only for SoftwareArtifacts?)

**Comment by jlovejoy** [comment]
@swinslow @zvr @goneall - made a few updates as per @zvr suggestion of removing "a" and other wordsmithing. I think there may be some other improvements on clarity of wording, but for now, this is good to go.

**Review by zvr — APPROVED** [review]

**Review by swinslow — APPROVED** [review]

**Comment by swinslow** [comment]
Thank you @jlovejoy and @zvr!

@goneall I think this is ready for your re-review and merging!

**Pull request merged**